content/module-reference/utility-modules/shared/collection-filters.md

```diff
@@ -0,0 +1,141 @@
+
+## range rating
+
+This new widget also allows you to select images by star rating, this time using a new "range" widget. The following image shows the widget with ratings of 2-4 stars selected.
+
+![range rating filter](./collection-filters/range-rating-filter.png#w33)
+
+You can choose a new range of ratings to filter by clicking and dragging over the widget's interface. You can also access pre-defined ranges by right-clicking and selecting from the popup menu. The number of applicable images is also shown against each entry in this menu.
+
+## range filters for numeric attributes
+
+Numeric attributes (aperture, exposure, focal length, iso, aspect ratio) are filtered using a widget that shows a histogram of the number of images available depending on the value of the given attribute (similar to the [_timeline_](../lighttable/timeline.md) in the lighttable view).
+
+For example, the following image shows the widget when used for selecting by aperture:
+
+![aperture range filter](./collection-filters/aperture-range-filter.png#w33)
+
+As with the range rating filter, you can select a range of values to filter by clicking and dragging on the widget. You can also access predefined ranges by right-clicking and selecting from the popup menu. The number of applicable images is also shown against each entry in this menu.
+
+You can also use the "min" and "max" text entry fields in the main module interface to manually define the bounds of the selection.
+
+## date attributes
+
+As with numeric and rating range filters, date/time filters are represented using a "range" widget. Select a range of values by clicking and dragging on the widget and use the right-click menu for more options.
+
+![date range filter](./collection-filters/capture-date-filter.png#w33)
+
+You can also use the "min" and "max" text entry fields in the main module interface to manually define the bounds of the selection (right-click for more options).
+
+The date/time format used by this module is `YYYY:MM:DD HH:MM:SS` (only the year values are mandatory).
+
+The keyword `now` is allowed in the max field (to define the current date/time).
+
+You can also use "relative" date/time values by preceding text entries with `+` or `-`. This allows you to set the maximum or minimum bound of the range relative to the other bound. As an example if you set the min value to `-0000:01` and the max value to `2022:04:15`, this will select images taken during the month before April 15th 2022.
+
+## filename
+
+The filename filter allows you to search images by their filename and/or extension. You can either enter the name or extension (with the leading`.`) manually or use the right-click menu to select.
+
+Within the extension field, you can also use the keywords `RAW` (to select all handled RAW file extensions), `NOT RAW` (to select all non-RAW file extensions), `LDR` (to select low-dynamic-range extensions) or `HDR` (to select high-dynamic-range extensions).
```

Comment:
Not sure (I've asked me the same the same for the tooltip in the widget itself) : do you think that it would be worth the pain, to explain what are LDR images (jpeg, png, ...) ??? This is imho the less common acronym here... but certainly the most used by peoples that don't know this part very well...

Reply:
TBH I'm not sure anyone will use this so I don't think it needs further explanation. We could link to the "supported file formats" section?

Reply:
well, to my knowledge, several people already use it (me included) but all are "experienced" users and LDR meaning is not a problem...
yes, why not linking to the "supported file format" section... but afaik this is not explained either... so maybe we should just let it like this...

Reply:
ok let's leave it like this for now and if we get any questions we can consider adding some further explanation. I assume you're ok with the rest of this now (I can merge)?